client/app/iplb/sslCertificate/iplb-ssl-certificate-edit.html

```diff
@@ -1,67 +1,51 @@
 <div class="cui-page__content">
-    <cui-back-link data-state="network.iplb.detail.ssl-certificate"
-                   data-state-params="{ serviceName: ctrl.$stateParams.serviceName}"
-                   data-text="'iplb_back_home' | translate"></cui-back-link>
-    <h2 class="oui-header_2" data-translate="iplb_ssl_add_title"></h2>
+    <oui-back-button data-title="{{'iplb_ssl_add_title' | translate }}"
+                     data-on-click="ctrl.$state.go('network.iplb.detail.ssl-certificate')">
+    </oui-back-button>
 
     <form novalidate name="ctrl.form">
-        <div class="oui-field">
-            <label for="displayName" class="oui-label" data-translate="iplb_ssl_edit_name"></label>
-            <div class="oui-field__content">
-                <div class="oui-field-control oui-field-control_4">
-                    <input type="text" id="displayName" name="displayName" class="oui-input" data-ng-model="ctrl.ssl.displayName" />
-                </div>
+        <oui-field label="{{'iplb_ssl_edit_name' | translate }}">
+            <div class="oui-field-control oui-field-control_4">
```

Comment:
it's too verbose and annoying to add it everywhere
 <oui-field size="sm"> could be more appropriate @neolitec.
I don't know for field with multiple controls..

Reply:
currently it is not possible to handle this without passing the class.